handlers/grafana_test.go

```diff
@@ -6,7 +6,8 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-	"k8s.io/api/core/v1"
+	core_v1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
why is the same import here twice with different aliases?

Reply:
good catch.